appoptics_metrics/alerts.py

```diff
@@ -56,23 +60,32 @@ def from_dict(cls, connection, data):
                   description=data['description'],
                   conditions=data['conditions'],
                   services=data['services'],
-                  _id=data['id'],
+                  _id=data.get('id'),
```

Comment:
Nice catch to use `.get()` to access the dict `data` instead of `.['blabla']` to avoid crashing. May you change all other accesses of `data` in this method accordingly?